### ISSUE TITLE
Handling of Paths for SrvD UserSubs Input Files

### ISSUE THREAD

**Issue by rdamiani** [OpenFAST/openfast#2059]
**Is your feature request related to a problem? Please describe.**

During Loads Analysis, it is convenient to leave Input and Parameter Files that do not change throughout the analysis in a "Template" or "Master" folder. This is the case, for example, of input files used by ServoDyn, such as `spd_trq.dat` and similar. 
Currently, `UserVSCont` is called with `p%RootName` as an argument, but that will seek `spd_trq.dat` in the "current" active folder of the `.fst`. Yet, the controller may not be changing across various DLC runs, and it would be efficient to leave it unaltered within a "template" folder thatcontains the main SrvD input file as well.

**Describe the solution you'd like**
 For this reason, `p%RootName` could be replaced by `p%PriPath` in the call to the UserSubs in SrvDyn. `PriPath`, path to the primary SrvD input file is already available, thus a small change to the registry would take care of this. I have implemented this change, and it seems to work out well. Alternatively, one would have to copy the `spd_trq` n_DLCs times in the various DLC folders...

**Comment by andrew-platt** [comment]
Thanks @rdamiani!  This is great suggestion.  Can you paste the code change here?  I can put this into a PR (with credit to you for the change) for our next release (v3.5.3).

**Comment by rdamiani** [comment]
Hi Andy, few changes indeed:

Around line 442 of the ServoDyn_Registry.txt:
````
typedef	^	ParameterType	CHARACTER(1024)	RootName	-	-	-	"RootName for writing output files"	-
typedef	^	ParameterType	CHARACTER(1024)	PriPath  	-	-	-	"Path of the primary SD input file "	-
````

And ServoDyn.f90 Around line 160:
````
CALL DispNVD( SrvD_Ver )
   CALL GetPath( InitInp%InputFile, PriPath )     ! Input files will be relative to the path where the primary input file is located.

   p%PriPath = PriPath
````
and then the calls to Usersubs such as around line 5990:

`CALL UserVSCont ( u%HSS_Spd, u%LSS_Spd, p%NumBl, t, p%DT, p%GenEff, 0.0_ReKi, p%PriPath, GenTrq, ElecPwr )`

I have also changed the following calls:
```
CALL UserGen ( u%HSS_Spd, u%LSS_Spd, p%NumBl, t, p%DT, p%GenEff, 0.0_ReKi, p%PriPath, GenTrq, ElecPwr )
CALL UserHSSBr ( y%GenTrq, y%ElecPwr, u%HSS_Spd, p%NumBl, t, p%DT, p%PriPath, HSSBrFrac )
CALL PitchCntrl ( u%BlPitch, ElecPwr, u%LSS_Spd, u%TwrAccel, p%NumBl, t, p%DT, p%PriPath, BlPitchCom )
CALL UserYawCont ( u%Yaw, u%YawRate, u%WindDir, u%YawErr, p%NumBl, t, p%DT, p%PriPath, YawPosCom, YawRateCom )
```

Best,
Rick

**Comment by andrew-platt** [comment]
Posted PR #2079 with these changes.

**Comment by andrew-platt** [comment]
Merged in these changes!  This will be included in the upcoming 3.5.3 release
